pants.ini

```diff
@@ -376,6 +376,10 @@ resolver_blacklist: {
   }
 
 
+[pytest]
+requirements: pytest>=3.0.7,<3.7
```

Comment:
Guessing the choice of putting this in `pants.ini` vs in the subsystem/options is conscious?

Reply:
Yes. This demonstrates dogfood of a fix for existing customers. If it works, HEAD can be fixed as you suggest - or with a fix of our custom node naming code in PytestRun.

Reply:
This fix works for me.

Reply:
Thanks @xinyuliu-glu. It does not appear to fix both Pants CI issues, but I may submit anyhow to trim things down to the remaining zip local header issue. Trying a local repro first.